index.js

```diff
@@ -0,0 +1,565 @@
+
+const ngrok = "https://74b76b4a.ngrok.io/";
+const filepath = "/Users/ek/Projects/dasmitdersilbe/";
+//const filepath = "/home/student/e/eukaiser/Projects/dasmitdersilbe/";
+const appId = "amzn1.ask.skill.f3bc5473-cad0-42f2-8e82-47664796986d";
+
+// ffmpeg -i <input-file> -ac 2 -codec:a libmp3lame -b:a 48k -ar 16000 <output-file.mp3>
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const Alexa = require('alexa-sdk');
+const _ = require('lodash');
+const fs = require('fs');
+const hf = require('./helper_functions.js');
+const data = require('./data.js');
+const db = require('./database.js')
+
+const app = express();
+
+app.use(bodyParser.json());
+
+app.get('/', function (req, res) {
+  //res.send('hello');
+  fs.createReadStream(filepath + 'test.mp3').pipe(res);
+});
+
+app.get('/1', function (req, res) { fs.createReadStream(filepath + '1.mp3').pipe(res); });
+app.get('/2', function (req, res) { fs.createReadStream(filepath + '2.mp3').pipe(res); });
+/*app.get('/3', function (req, res) { fs.createReadStream(filepath + '3.mp3').pipe(res); });
+app.get('/4', function (req, res) { fs.createReadStream(filepath + '4.mp3').pipe(res); });
+app.get('/5', function (req, res) { fs.createReadStream(filepath + '5.mp3').pipe(res); });
+app.get('/6', function (req, res) { fs.createReadStream(filepath + '6.mp3').pipe(res); });
+app.get('/7', function (req, res) { fs.createReadStream(filepath + '7.mp3').pipe(res); });
+app.get('/8', function (req, res) { fs.createReadStream(filepath + '8.mp3').pipe(res); });*/
+app.get('/a', function (req, res) { fs.createReadStream(filepath + 'geburtstagstanz.mp3').pipe(res); });
+app.get('/b', function (req, res) { fs.createReadStream(filepath + 'zau_zaubert.mp3').pipe(res); });
+app.get('/c', function (req, res) { fs.createReadStream(filepath + 'schneller_teller.mp3').pipe(res); });
+
+app.post('/', function (req, res) {
+  let context = {
+    succeed: function (result) {
+      res.json(result);
+    },
+    fail: function (error) {
+      console.log(error);
+    }
+  };
+  let alexa = Alexa.handler(req.body, context);
+
+  alexa.registerHandlers(
+    handlers,
+    start_handlers,
+    lection_handlers,
+    music_handlers,
+    game_handlers
+  );
+
+  alexa.appId = appId;
+  alexa.execute();
+});
+
+let server = app.listen(process.env.PORT || 4567, function () {
+  let host = server.address().address;
+  let port = server.address().port;
+  console.log("Server listening at http://%s:%s", host, port);
+  console.log("");
+});
+
+const handlers = {
+  "LaunchRequest": function () {
+    db.addUser(this.event.session.user.userId);
+
+    // USE ONCE AT DATABASE CREATION
+    //hf.createAllLectures();
```

Comment:
Do shouldn't expect users to read all the code before launching the app for the first time.
Or is this not required anymore?

Reply:
It is still in use, but I need a different workaround for creating the lectures once.